cordova-lib/src/cordova/lazy_load.js

```diff
@@ -147,10 +147,7 @@ function cordova_npm(platform) {
 // Returns a promise that resolves to directory containing the package.
 function npm_cache_add(pkg) {
     var npm_cache_dir = path.join(util.libDirectory, 'npm_cache');
-    // 'cache-min' is the time in seconds npm considers the files fresh and
```

Comment:
Consider totally removing calls to 'npm cache add'. Also, are we sure that npm doesn't set some kind of default cache-min ?